src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -462,8 +462,12 @@ int ZYNQShim::xclLoadAxlf(const axlf *buffer)
 	}
 
 #if defined(XCLBIN_DOWNLOAD)
-  drm_zocl_pcap_download obj = { const_cast<axlf *>(buffer) };
-  ret = ioctl(mKernelFD, DRM_IOCTL_ZOCL_PCAP_DOWNLOAD, &obj);
+	drm_zocl_pcap_download obj = { const_cast<axlf *>(buffer) };
+	ret = ioctl(mKernelFD, DRM_IOCTL_ZOCL_PCAP_DOWNLOAD, &obj);
```

Comment:
Why we still need DRM_IOCTL_ZOCL_PCAP_DOWNLOAD?
Download bitstream in DRM_IOCTL_ZOCL_READ_AXLF, if BITSTREAM section is found.